proposals/0251-font-styles.md

```diff
@@ -28,30 +28,55 @@ The HMI should be able to provide a list of supported hyper text elements to SDL
 ```xml
 <struct name="TextField" since="1.0">
 :
-    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minvalue="1" maxvalue="100" since="6.x">    
+    <param name="hyperTextSupported" type="String" array="true" mandatory="false" minsize="1" maxsize="100" since="6.x">
+        <description>A list of supported html tags without the < > characters.</description>
     </param>
 </struct>
 ```
 
 The HMI API equivalent should exclude the versioning attributes.
 
+The head unit would just send an array of html tags without the < > characters. For a list of current html tags see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element#Inline_text_semantics
+For example:
+```json
+"hyperTextSupported":[ "b", "u", "i" ]
+```
+
 Alternative: An enum can be added that lists all the hyper text elements.
 
-With this implementation, technically the HTML tags take up part of the 500 character count restriction placed in `TextFieldStruct`. So, the character count for `TextFieldStruct` would go from 500 to 1000 in the HMI API. This struct is used for pretty much any text field in RPCs like `Alert`, `ScrollableMessage`, `Show` and so on.
+With this implementation, technically the HTML tags take up part of the 500 character count restriction placed in `TextFieldStruct`. So, the character count for `TextFieldStruct` would go from 500 to 1000 in the HMI API. This struct is used for pretty much any text field in RPCs like `Alert`, `ScrollableMessage`, `Show` and so on. This adds some complexity for older headunits. If after all html tags have been stripped from the text and the text is still over 500 characters than the mobile library needs to send back the response of `INVALID_DATA`.
 ```xml
 <struct name="TextFieldStruct">
   .
   .
   .
   <param name="fieldText" type="String" maxlength="1000" mandatory="true">
     <description>The text itself.</description>
+    <history>
```

Comment:
This struct is only in the HMI API, and the HMI API does not use the history tag, only the MOBILE_API. The history tag can be removed from this struct param.